src/components/LandingFooter/index.js

```diff
@@ -114,6 +115,13 @@ const LandingFooter = () => (
                   >
                     Blog
                   </a>
+                  <a
+                    href="https://github.com/cranecloud.io"
```

Comment:
@mosesmulumba replace that with  https://github.com/crane-cloud

Reply:
isn't blog supposed to be on a different line than contribute @mosesmulumba

Reply:
![image](https://user-images.githubusercontent.com/69305400/189209048-6d11fb6a-6726-4db1-a5d6-be2b2f068f65.png)